.gitlab-ci.yml

```diff
@@ -104,3 +107,49 @@ test_image:
 
     - podman run $tag
     - podman logout $QUAY_HOST
+
+Spack Build:
+  stage: build
+  before_script:
+    - whoami
+    - mkdir /usr/workspace/morton30/spack/
```

Comment:
Do we have the option of using something like `$TMP`?  Otherwise this recipe will only work for your account I assume?

Reply:
Yes, that's true. I just requested a new LC group, so I can change the directory to `/usr/workspace/exaworks/sdk/spack/` and anyone within the exaworks group will be able to run.